core/storage/user/gae_models_test.py

```diff
@@ -21,6 +21,23 @@
 (user_models,) = models.Registry.import_models([models.NAMES.user])
 
 
+class UserSettingsModelTest(test_utils.GenericTestBase):
+    """Tests for the UserSettingsModel class."""
+    USER_NAME = 'test_user'
+    USER_EMAIL = 'test@email.com'
+
+    def setUp(self):
+        super(UserSettingsModelTest, self).setUp()
+        user_models.UserSettingsModel(
+            email=self.USER_EMAIL, username=self.USER_NAME).put()
+
+    def test_user_gets_correct_role_by_default(self):
+        user = user_models.UserSettingsModel.get_all().filter(
```

Comment:
This is too complicated for this test. Why not just get the relevant user by ID?